dist/api_data.py

```diff
@@ -151,6 +151,14 @@ def __ge__(self, other):
                 Conversely, if the delta came to 21 bytes, reconciliation would not emit a
                 delta. Deltas larger than full pages are permitted for measurement and testing
                 reasons, and may be disallowed in future.''', min='1', max='1000'),
+            Config('internal_page_delta', 'false', r'''
```

Comment:
I'm a bit inclined to put these configuration options somewhere else. Config options that can appear in tables tend to be "sticky" in that they appear in metadata that gets persisted.

How would you feel about making this a connection wide configuration in the connection_disaggregated_config_common section instead?

Reply:
I also have a slight preference for putting the config in at the connection level.